{"uid":"ef7c905e0d169812","name":"test_ingress_integration","fullName":"tests.integration.legacy.test_ingress#test_ingress_integration","historyId":"e489d4376139a099c9882e87d9bf9aa2","time":{"start":1761958869312,"stop":1761958869312,"duration":0},"description":"Deploy the charm with anycharm ingress requirer that installs apache2.\n\n    Assert that the requirer endpoint is available.\n    ","descriptionHtml":"<p>Deploy the charm with anycharm ingress requirer that installs apache2.</p>\n<pre><code>Assert that the requirer endpoint is available.\n</code></pre>\n","status":"broken","statusMessage":"TimeoutError: Timed out waiting for model:\n  any-charm-ingress-requirer/0 [allocating] waiting: waiting for machine","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 843, in pytest_runtest_setup\n    yield\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 895, in pytest_runtest_setup\n    return (yield)\n            ^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 164, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1674, in setup\n    self._request._fillfixtures()\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 720, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 549, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 640, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1128, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 930, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 302, in _asyncgen_fixture_wrapper\n    result = event_loop.run_until_complete(setup())\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 284, in setup\n    res = await gen_obj.__anext__()\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/tests/integration/legacy/conftest.py\", line 356, in any_charm_ingress_requirer_fixture\n    await model.wait_for_idle(apps=[application.name], status=\"active\")\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 3250, in wait_for_idle\n    raise asyncio.TimeoutError(\nTimeoutError: Timed out waiting for model:\n  any-charm-ingress-requirer/0 [allocating] waiting: waiting for machine","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1761958869313,"stop":1761958869313,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1761958869313,"stop":1761958869313,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"pytestconfig","time":{"start":1761958869313,"stop":1761958869313,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1761958869313,"stop":1761958869711,"duration":398},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"charm","time":{"start":1761958869313,"stop":1761958869313,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"application","time":{"start":1761958869711,"stop":1761958939123,"duration":69412},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"model","time":{"start":1761958869711,"stop":1761958869711,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"certificate_provider_application","time":{"start":1761958939123,"stop":1761958940523,"duration":1400},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"configured_application_with_tls","time":{"start":1761958940523,"stop":1761959009011,"duration":68488},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"any_charm_src_ingress_requirer","time":{"start":1761959009011,"stop":1761959009012,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1761959009012,"stop":1761959009012,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"any_charm_ingress_requirer_name","time":{"start":1761959009012,"stop":1761959009012,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"any_charm_ingress_requirer","time":{"start":1761959009012,"stop":1761959612024,"duration":603012},"status":"broken","statusMessage":"TimeoutError: Timed out waiting for model:\n  any-charm-ingress-requirer/0 [allocating] waiting: waiting for machine\n","statusTrace":"  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 930, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 302, in _asyncgen_fixture_wrapper\n    result = event_loop.run_until_complete(setup())\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 284, in setup\n    res = await gen_obj.__anext__()\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/tests/integration/legacy/conftest.py\", line 356, in any_charm_ingress_requirer_fixture\n    await model.wait_for_idle(apps=[application.name], status=\"active\")\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 3250, in wait_for_idle\n    raise asyncio.TimeoutError(\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"description":"Deploy the charm with anycharm ingress requirer that installs apache2.\n\n    Assert that the requirer endpoint is available.\n    ","status":"broken","statusMessage":"TimeoutError: Timed out waiting for model:\n  any-charm-ingress-requirer/0 [allocating] waiting: waiting for machine","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 843, in pytest_runtest_setup\n    yield\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 895, in pytest_runtest_setup\n    return (yield)\n            ^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 164, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1674, in setup\n    self._request._fillfixtures()\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 720, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 549, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 640, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1128, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 930, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 302, in _asyncgen_fixture_wrapper\n    result = event_loop.run_until_complete(setup())\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 284, in setup\n    res = await gen_obj.__anext__()\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/tests/integration/legacy/conftest.py\", line 356, in any_charm_ingress_requirer_fixture\n    await model.wait_for_idle(apps=[application.name], status=\"active\")\n  File \"/home/ubuntu/actions-runner/_work/haproxy-operator/haproxy-operator/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 3250, in wait_for_idle\n    raise asyncio.TimeoutError(\nTimeoutError: Timed out waiting for model:\n  any-charm-ingress-requirer/0 [allocating] waiting: waiting for machine","steps":[],"attachments":[{"uid":"1615a756ea86d3ad","name":"log","source":"1615a756ea86d3ad.txt","type":"text/plain","size":6714}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1761959612029,"stop":1761959612029,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"application::finalizer","time":{"start":1761959612030,"stop":1761959612030,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1761959612030,"stop":1761959612400,"duration":370},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1761959612400,"stop":1761959612400,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1761959612400,"stop":1761959612400,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1761959612400,"stop":1761959612401,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.legacy"},{"name":"suite","value":"test_ingress"},{"name":"host","value":"github-runner"},{"name":"thread","value":"13850-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.legacy.test_ingress"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"8024eff89ce9ef7a","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[]}],"history":{"statistic":{"failed":0,"broken":6,"skipped":0,"passed":0,"unknown":0,"total":6},"items":[{"uid":"62eae4165791b11c","reportUrl":"../997//#testresult/62eae4165791b11c","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\n  any-charm-ingress-requirer/0 [allocating] waiting: waiting for machine","time":{"start":1761572962236,"stop":1761572962236,"duration":0}},{"uid":"64614af59bd4765f","reportUrl":"../996//#testresult/64614af59bd4765f","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\n  any-charm-ingress-requirer/0 [allocating] waiting: waiting for machine","time":{"start":1761570778546,"stop":1761570778546,"duration":0}},{"uid":"16c4ef7dfd3776ab","reportUrl":"../995//#testresult/16c4ef7dfd3776ab","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\n  any-charm-ingress-requirer/0 [allocating] waiting: waiting for machine","time":{"start":1761563918352,"stop":1761563918352,"duration":0}},{"uid":"836e91070eb97776","reportUrl":"../994//#testresult/836e91070eb97776","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\n  any-charm-ingress-requirer/0 [allocating] waiting: waiting for machine","time":{"start":1761306511345,"stop":1761306511345,"duration":0}},{"uid":"9081fa5078b66aca","reportUrl":"../991//#testresult/9081fa5078b66aca","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\n  any-charm-ingress-requirer/0 [allocating] waiting: waiting for machine","time":{"start":1761042525477,"stop":1761042525477,"duration":0}}]},"tags":["asyncio","abort_on_fail"]},"source":"ef7c905e0d169812.json","parameterValues":[]}